holidays/countries/marshall_islands.py

```diff
@@ -13,78 +13,90 @@
 from datetime import date
 from datetime import timedelta as td
 
-from dateutil.easter import easter
-
-from holidays.calendars import _get_nth_weekday_from, _get_nth_weekday_of_month
-from holidays.constants import JAN, MAR, MAY, JUL, SEP, NOV, DEC, MON, FRI
+from holidays.calendars import _get_nth_weekday_of_month
+from holidays.constants import MAR, MAY, JUL, SEP, NOV, DEC, FRI
 from holidays.holiday_base import HolidayBase
+from holidays.holiday_groups import ChristianHolidays, InternationalHolidays
 
 
-class HolidaysMH(HolidayBase):
+class HolidaysMH(HolidayBase, ChristianHolidays, InternationalHolidays):
     """
     https://rmiparliament.org/cms/component/content/article/14-pressrelease/49-important-public-holidays.html?Itemid=101
     https://www.rmiembassyus.org/country-profile#:~:text=national%20holidays
     """
 
     country = "MH"
+
+    # Special Holidays
+
+    # General Election Day
+    election_day = "General Election Day"
+
     special_holidays = {
-        2019: (NOV, 18, "General Election Day"),  # Third Monday in November
```

Comment:
If we know the rule for this holiday, we can make it common, not special, like:
```python
        # General Election Day
        if year >= 1995 and (year - 1995) % 4 == 0:
            self._add_holiday("General Election Day", _get_nth_weekday_of_month(3, MON, NOV, year))

```

Reply:
Fine by me, though these events are infrequent enough that special holidays work. On top of that, who knows if these elections will get postponed or held earlier via snap elections in the future.

Reply:
Turns out the 1999 ones took place on the 4th Monday (NOV 22) rather than the 3rd (NOV 15) so the actual code would look like this:
```python
        if year >= 1995 and (year - 1995) % 4 == 0:
            self._add_holiday(
                # General Election Day
                "General Election Day",
                date(year, NOV, 22) if year == 1999 
                else self._get_nth_weekday_of_month(3, MON, NOV)
            )
```
Lemme know if you prefer this approach or whether sticking to the special holidays works better @KJhellico

Reply:
Let's leave it as a special holidays :)